docs/index.rst

```diff
@@ -36,6 +36,7 @@ The Relay takes two optional arguments:
    :glob:
 
    overview.rst
+   time_alignment.rst
```

Comment:
Please move overview and time alignment to a `Guides` subfolder, similar to https://github.com/pupil-labs/realtime-python-api/blob/main/docs/index.rst